shared/src/models/utils/occupations/occupation-helpers-mutable.ts

```diff
@@ -0,0 +1,28 @@
+import type { Mutable } from '../../../utils';
+import { cloneDeepMutable } from '../../../utils';
+import type { ExerciseOccupation } from './exercise-occupation';
+import { NoOccupation } from './no-occupation';
+
+export function isUnoccupied(
+    occupied: Mutable<{ occupation: ExerciseOccupation }>,
```

Comment:
Please also add functions for the case when we do not have a `Mutable` at hand.

Reply:
2b95c61fffc46093fbdc363bb2b0e79f5b12b967